DEVELOPMENT.md

```diff
@@ -90,9 +90,13 @@ By default KubeArchive listens to `Event`s in the `test` namespace.
     ```bash
     kubectl port-forward -n kubearchive svc/kubearchive-api-server 8081:8081
     ```
+1. Get the Certificate Authority (CA) from the `kubearchive-api-server-tls` secret:
+    ```bash
+    kubectl get -n kubearchive secrets kubearchive-api-server-tls -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
```

Comment:
could you add '$' at the beginning of the command?

Reply:
for code blocks like this, github will generate a button to copy all of the text. If the `$` is added at the beginning, you have to go to the beginning of the line to remove the `$` if you paste the command into your terminal which I think is kind of annoying

Reply:
That's it, I'm keeping the commands without dollar sign so they are easier to copy.